src/components/settings/DataManagement/index.tsx

```diff
@@ -1,40 +1,124 @@
 import { useState } from 'react'
-import { Paper, Grid, Typography, Button, Link } from '@mui/material'
-import Track from '@/components/common/Track'
-import { SETTINGS_EVENTS } from '@/services/analytics'
-import ImportAllDialog from '../ImportAllDialog'
+import { Paper, Grid, Typography, Button, SvgIcon, Box } from '@mui/material'
+
+import FileIcon from '@/public/images/settings/data/file.svg'
+import ExportIcon from '@/public/images/common/export.svg'
+import { getPersistedState, useAppSelector } from '@/store'
+import { addressBookSlice, selectAllAddressBooks } from '@/store/addressBookSlice'
+import { addedSafesSlice, selectAllAddedSafes } from '@/store/addedSafesSlice'
+import { safeAppsSlice, selectSafeApps } from '@/store/safeAppsSlice'
+import { selectSettings, settingsSlice } from '@/store/settingsSlice'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import ExternalLink from '@/components/common/ExternalLink'
+import { ImportFileUpload } from '@/components/settings/DataManagement/ImportFileUpload'
+import { ImportDialog } from '@/components/settings/DataManagement/ImportDialog'
+import { SAFE_EXPORT_VERSION } from '@/components/settings/DataManagement/useGlobalImportFileParser'
+import { FileListCard } from '@/components/settings/DataManagement/FileListCard'
+
+import css from './styles.module.css'
+
+const getExportFileName = () => {
+  const today = new Date().toISOString().slice(0, 10)
+  return `safe-${today}.json`
+}
+
+export const handleExport = () => {
```

Comment:
nit: I would give this a more specific name since its being exported. Maybe something like `exportAppData`?